hack/test-pj.sh

```diff
@@ -1,28 +1,55 @@
 #!/usr/bin/env bash
 # Inspired by https://github.com/kubernetes/test-infra/blob/047b0a0596f7d42b909405c0b8ed4366de2bef72/prow/pj-on-kind.sh
 # Required kubectl
-# Rightnow this work only on the x86 platform(Linux, MacOS)
+# Right now this work only on the x86 platform(Linux, MacOS)
 
 set -o errexit
-set -o nounset
 set -o pipefail
```

Comment:
As this script has many variables involved, I suppose having `set -o nounset` enabled would prevent unexpected undefined variable errors and force the script to exit immediately if any undefined variables are encountered.
You may define a default empty value for the purpose variable instead of removing this.

Reply:
@AshwinHIBM Are you not addressing this?

Reply:
I have addressed it in the latest version of the file here (in this pull request itself): https://github.com/ppc64le-cloud/test-infra/pull/379/commits/63b8ebb2ee507d421edb77321779090f9153caf9#diff-cd93e43749b2cc1b84519bd1469e4e5d05dbd6780fb5be96f0606986dd03652e

It's showing the old version of the file here because your comment was applicable to that version.